io.github.fabiangreffrath.Doom.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "io.github.fabiangreffrath.Doom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "crispy-doom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=x11",
```

Comment:
```suggestion
```
You don't need x11 if it can run under wayland, so does it really? if not, you should remove fallback-x11

Reply:
Thanks.  I wasn't sure from the documentation, so I cribbed from the GZDoom manifest.

It does run under Wayland, so I've removed this line.